c++/src/sargs/PredicateLeaf.cc

```diff
@@ -390,10 +390,15 @@ namespace orc {
 
   DIAGNOSTIC_POP
 
+  static bool colHasNullForwardCompatible(const proto::ColumnStatistics& stats) {
```

Comment:
Why calling it `forward compatible`? It seems to be a bug from a legacy writer IMHO.